instances/widgets.treasury-factory.near/widget/components/templates/AppLayout.jsx

```diff
@@ -301,14 +301,90 @@ const Theme = styled.div`
     .amount {
       font-size: 14px;
     }
+
+    tbody {
+      tr {
```

Comment:
please make sure it works fine with tables in existing places, there's quite a few.